app/api/v2/handlers/fact_api.py

```diff
@@ -27,10 +27,11 @@ def add_routes(self, app: web.Application):
         router.add_patch('/facts', self.update_facts)
         router.add_patch('/relationships', self.update_relationships)
 
-    @aiohttp_apispec.docs(tags=['facts'], summary="Retrieve facts by criteria. Use fields from the FactSchema "
-                                                  "in the request body to filter retrieved facts.")
+    @aiohttp_apispec.docs(tags=['facts'],
+                          summary='Retrieve Facts',
+                          description='Retrieve facts by criteria. Use fields from the `FactSchema` in the request body to filter retrieved facts.')
     @aiohttp_apispec.querystring_schema(BaseGetAllQuerySchema)
-    @aiohttp_apispec.response_schema(FactSchema(many=True, partial=True))
+    @aiohttp_apispec.response_schema(FactSchema(many=True, partial=True), description='Returns facts in `FactSchema` format.')
```

Comment:
For this, it might be helpful for users just to change it to something like `Returns matching facts in `FactSchema` format.`